Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -514,6 +521,34 @@ static void activate(GApplication* application, WPEToolingBackends::ViewBackend*
 #endif
 }
 
+static gboolean parseWindowSize()
+{
+    if (!windowSize)
+        return FALSE;
+
+    gchar** windowSizes = g_strsplit(windowSize, "x", 2);
+    if (!windowSizes || !windowSizes[0] || !windowSizes[1]) {
+        g_warning("Failed to parse --size command line argument - use \"<size>x<height>\" without any additional whitespace.");
+        return FALSE;
+    }
+
+    guint64 parsedWidth = 0;
+    if (!g_ascii_string_to_unsigned(windowSizes[0], 10, 0, G_MAXUINT, &parsedWidth, nullptr)) {
+        g_warning("Failed to parse window width as unsigned integer from string \"%s\"", windowSizes[0]);
+        return FALSE;
+    }
+
+    guint64 parsedHeight = 0;
+    if (!g_ascii_string_to_unsigned(windowSizes[1], 10, 0, G_MAXUINT, &parsedHeight, nullptr)) {
+        g_warning("Failed to parse window height as unsigned integer from string \"%s\"", windowSizes[1]);
+        return FALSE;
+    }
+
+    windowWidth = static_cast<guint>(parsedWidth);
+    windowHeight = static_cast<guint>(parsedHeight);
```

Comment:
We should probably check that both are greater than 0.

Reply:
Indeed, will adapt.